FE/src/components/Account.jsx

```diff
@@ -1,45 +1,51 @@
-import {useEffect, useState } from 'react'
-import { useNavigate } from 'react-router-dom'
+import {useEffect, useState} from 'react'
+import { Routes, Route, useNavigate, useParams} from 'react-router-dom'
+import { Link } from 'react-router-dom'
+import UserInfo from './UserInfo'
```

Comment:
I would put these all in one folder and destructure these on import. It would make it a TON cleaner.